### PR TITLE
docs(ecocredit): update client docs

### PR DESCRIPTION
**Pull request by aleem1314**
## Description

Ref: #931 

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed 
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: x/ecocredit/spec/07_client.md [hunk 1]
```diff
@@ -6,806 +6,1002 @@ A user can query and interact with the `ecocredit` module using the CLI.
...
-regen query ecocredit types [flags]
+regen.ecocredit.v1.Query/ProjectsByAdmin
 ```
 
 Example:
 
 ```bash
-regen query ecocredit types
+grpcurl -plaintext \
+    -d '{"admin":"regen1v2ncquer9r2ytlkxh2djmmsq3e8we6rjc9snfn"}'
```
Comment (aleem1314) [inline x/ecocredit/spec/07_client.md:273]:
```suggestion
    -d '{"admin":"regen1v2ncquer9r2ytlkxh2djmmsq3e8we6rjc9snfn"}' \
```

File: x/ecocredit/spec/07_client.md [hunk 2]
```diff
@@ -6,806 +6,1002 @@ A user can query and interact with the `ecocredit` module using the CLI.
...
-regen query ecocredit projects [class_id] [flags]
+regen.ecocredit.v1.Query/ProjectsByClass
 ```
 
 Example:
 
 ```bash
-regen query ecocredit projects C01
+grpcurl -plaintext \
+    -d '{"class_id":"C01"}'
```
Comment (aleem1314) [inline x/ecocredit/spec/07_client.md:195]:
```suggestion
    -d '{"class_id":"C01"}' \
```

File: x/ecocredit/spec/07_client.md [hunk 3]
```diff
@@ -6,806 +6,1002 @@ A user can query and interact with the `ecocredit` module using the CLI.
...
-regen query ecocredit supply [batch_denom] [flags]
+regen.ecocredit.v1.Query/ProjectsByReferenceId
 ```
 
 Example:
 
 ```bash
-regen query ecocredit supply C01-20200101-20210101-001
+grpcurl -plaintext \
+    -d '{"reference_id":"VERRA01"}'
```
Comment (aleem1314) [inline x/ecocredit/spec/07_client.md:233]:
```suggestion
    -d '{"reference_id":"VERRA01"}' \
```

File: x/ecocredit/spec/07_client.md [hunk 4]
```diff
@@ -814,387 +1007,468 @@ Example:
...
+      "admin": "regen1v2ncquer9r2ytlkxh2djmmsq3e8we6rjc9snfn",
+      "class_id": "C01",
+      "jurisdiction": "KE",
+      "metadata": "",
+      "reference_id": ""
+    }
+  ],
+  "pagination": {
+    "next_key": null,
+    "total: 2
```
Comment (ryanchristo) [inline x/ecocredit/spec/07_client.md:1351]:
```suggestion
    "total": "2"
```

File: x/ecocredit/spec/07_client.md [hunk 5]
```diff
@@ -814,387 +1007,468 @@ Example:
...
+      "class_id": "C01",
+      "jurisdiction": "KE",
+      "metadata": "",
+      "reference_id": "R01"
     }
   ],
   "pagination": {
-    "total": "1"
+    "next_key": null,
+    "total: 2
```
Comment (ryanchristo) [inline x/ecocredit/spec/07_client.md:1262]:
```suggestion
    "total": "2"
```

**Review by ryanchristo — COMMENTED** [review]
Thanks for taking this on! This is looking good. I left a handful of minor suggestions. Let's make sure we are using the updated batch denom in the examples

**Review by ryanchristo — APPROVED** [review]
Looks good to me. :+1:

**Review by technicallyty — APPROVED** [review]

**Pull request merged**